src/__tests__/gatsby-node.tests.js

```diff
@@ -3,12 +3,12 @@ import {
   createContentDigest,
   makeNavigationNode,
   makeBreadcrumbsNode,
+  ploneNodeGenerator,
   fetchPloneNavigationNode,
   fetchPloneBreadcrumbsNode,
 } from '../utils';
-
 // This import is here only to make gatsby-node to appear in overall coverage
-import { ploneNodeGenerator } from '../gatsby-node';
+// import { ploneNodeGenerator } from '../gatsby-node';
```

Comment:
I believe this should still remain imported to allow gatsby-node to be included in overall coverage count

Reply:
I am resolving this because you already made the changes.